packages/vue-split-panel/src/SplitPanel.vue

```diff
@@ -21,6 +21,10 @@ const props = withDefaults(defineProps<SplitPanelProps>(), {
 	snapThreshold: 12,
 });
 
+const emits = defineEmits<{
+	(e: 'transitionend'): void;
+}>();
```

Comment:
Lets use the Vue 3.3+ syntax here :) 


```suggestion
const emits = defineEmits<{
	transitionend: []
}>();
```